$MirOS: ports/devel/pkgconfig/patches/patch-pkg-config_1,v 1.1 2008/12/26 15:42:06 tg Exp $
--- pkg-config.1.orig	Wed Jan 16 22:26:27 2008
+++ pkg-config.1	Fri Nov 20 18:24:13 2009
@@ -53,7 +53,7 @@ program: program.c
 \fIpkg-config\fP retrieves information about packages from 
 special metadata files. These files are named after the package, 
 with the extension \fI.pc\fP. By default, pkg-config looks in 
-the directory \fIprefix\fP/lib/pkgconfig for these files; it will also
+the directory /usr/libdata/pkgconfig for these files; it will also
 look in the colon-separated (on Windows, semicolon-separated) 
 list of directories specified by the 
 PKG_CONFIG_PATH environment variable. 
